### DISCUSSION TITLE
The `fill` function explanation is unclear when we say handle out of bounds, negative values

### DISCUSSION THREAD

**Discussion by devmin8** [greatfrontend/greatfrontend#49] [Feedback/Bug Reports]
Also the test case description says `fills array with value from start to end when out of range indices are provided` but the values are like 

```
const arr = [1, 2, 3];
expect(fill(arr, '*', 1, 10)).toEqual([1, '*', '*', '*']);
```

Here not sure why the `*` in the index `3` in the output array.

**Comment by faustaleonardo** [comment]
Agree. Quite confusing. Plus, not sure why there is no solution available yet.

**Comment by yangshun** [comment]
Sorry for the confusion, it has been fixed.
